secureboot_policy.go

```diff
@@ -991,9 +992,9 @@ func (g *secureBootPolicyGen) computeAndExtendVerificationMeasurement(paths []*s
 		var digests *tpm2.DigestList
 		switch source {
 		case Firmware:
-			digests = &p.firmwareVerificationEvents
+			digests = &b.firmwareVerificationEvents
 		case Shim:
-			digests = &p.shimVerificationEvents
+			digests = &b.shimVerificationEvents
 		}
 		measured := false
 		for _, d := range *digests {
```

Comment:
this code here feels like it should be a predicate on *Branch, maybe there's other similar things

Reply:
I've moved a bunch of this code to secureBootPolicyGenBranch now.